examples/out-of-memory/build.rs

```diff
@@ -16,6 +16,11 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
+use gear_wasm_builder::WasmBuilder;
+
 fn main() {
-    gear_wasm_builder::build();
+    WasmBuilder::new()
+        .exclude_features(vec!["std"])
+        .with_forced_recommended_toolchain()
```

Comment:
I'd add comment about internal usage because this demo is in `examples`

Reply:
beb4d72dacad8c348c42d5ea1ea7a1a1540efea9